docs/tasks/stateful-sets/debugging-a-statefulset.md

```diff
@@ -0,0 +1,78 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to debug a Stateful Set.
```

Comment:
Don't forget to update to CamelCase as per https://github.com/kubernetes/kubernetes.github.io/pull/1807.